istioctl/cmd/istioctl/main.go

```diff
@@ -96,10 +96,13 @@ system.
 
 Available routing and traffic management configuration types:
```

Comment:
If you are providing the whole list, then this should just say "Available configuration types:"